app/containers/LanguageProvider/__tests__/LanguageProvider.test.js

```diff
@@ -33,7 +33,8 @@ describe('<ConnectedLanguageProvider />', () => {
   let store;
 
   beforeAll(() => {
-    store = configureStore({}, browserHistory);
+    // eslint-disable-next-line prefer-destructuring
+    store = configureStore({}, browserHistory).store;
```

Comment:
Or `({ store } = configureStore({}, browserHistory))`. Then you can remove the eslint disable line.

Reply:
Yes that much better, fixed this and all subsequent occurrences.